killerbee/dev_apimote.py

```diff
@@ -259,18 +259,22 @@ def ping(self, da: Any, panid: Any, sa: Any, channel: Optional[int]=None, page:
         '''
         raise Exception('Not yet implemented')
 
-    def jammer_on(self, channel: Optional[int]=None, page: int=0) -> None:
+    def jammer_on(self, channel: Optional[int]=None, page: int=0, method: Optional[str]=None) -> None:
         '''
-        Not yet implemented.
+        Implements reflexive jamming or constant carrier wave jamming.
         @type channel: Integer
         @param channel: Sets the channel, optional
         @type page: Integer
         @param page: Sets the subghz page, not supported on this device
         @rtype: None
         '''
+
         if self.handle is None:
             raise Exception("Handle does not exist")
 
+        if method is None or method not in ['reflexive', 'constant']:
+            raise Exception("Device does not support the provided jamming method")
```

Comment:
Probably should be ValueError, or leverage the `self.capabilities.require(KBCapabilities.PHYJAM_REFLEX)` type checks.

Reply:
I just copied from your PR feel free to make any changes you want